docs/langref/src/concepts/file.md

```diff
@@ -63,12 +70,12 @@ export component MyApp inherits Window {
 }
 ```
 
-Names have to be valid [identifiers](identifiers.md).
+Names have to be valid [identifiers](../identifiers.md).
 
 Some elements are also accessible under pre-defined names:
 
 -   `root` refers to the outermost element of a component.
 -   `self` refers to the current element.
 -   `parent` refers to the parent element of the current element.
 
-These names are reserved and can't be re-defined by the user.
+Users can not redefine these names to refer to something else.
```

Comment:
How about something even simpler (and using a contraction). This way the emphasis is on the fact that the names are reserved, and the consequence follows after that (not being possible to re-defined), instead of omitting the reason why the user can't re-define them (they are reserved).

```suggestion
These names are reserved and you can't be re-define them.
```